src/services/accommodation.service.js

```diff
@@ -0,0 +1,55 @@
+import models from '../models';
+
+const { Accommodation, AccommodationType } = models;
+
+/**
+ *
+ *
+ * @class AccommodationService
+ */
+class AccommodationService {
+  /**
+         *
+         *
+         * @static
+         * @param {newAccommodation} newAccommodation
+         * @returns {newAccommodation} @memberof AccommodationService
+         */
+  static async createAccommodation(newAccommodation) {
+    const { dataValues } = await Accommodation.create(newAccommodation);
+    const { updatedAt, createdAt, ...newAccommodationData } = dataValues;
```

Comment:
Is there a specific reason why you are filtering out the date columns? Usually, you will need this on the frontend to show when the accommodation was created/updated.